schedule/jeos/sle/rpi/jeos-realhw.yaml

```diff
@@ -8,6 +8,7 @@ schedule:
   - console/ntp_client
   - console/system_prepare
   - console/force_scheduled_tasks
+  - console/journal_check
```

Comment:
Could it make sense to run this after the wifi and bluetooth test to cover error messages due to wifi/bt operation?

Reply:
done